python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -1854,6 +1861,9 @@ def test_dataframe_min_count_ops(data, ops, skipna, min_count):
     psr = pd.DataFrame(data)
     gsr = gd.DataFrame(data)
 
+    if psr.shape[0] * psr.shape[1] < min_count:
+        pytest.xfail("https://github.com/pandas-dev/pandas/issues/39738")
```

Comment:
Should we version this?

Reply:
Versioned all the pandas related issues 👍